dot-net-sdk/helpers/AppDetails.cs

```diff
@@ -4,47 +4,51 @@ namespace eppo_sdk.helpers;
 
 public class AppDetails
 {
-    private const string SDK_LANG = "c#";
```

Comment:
dropping sdkLanguage as it's not logged anyway.

Reply:
FWIW, In other SDKs we log it as part of the metadata in the assignment event.

Reply:
Never mind I'm commenting too quickly as usual

Reply:
*Not logged by the CDN is what I should have said here.